packages/app/src/runner/event-manager.ts

```diff
@@ -611,15 +607,19 @@ export class EventManager {
         })
       }
 
-      await Cypress.backend('protocol:test:before:run:async', { id: test.id, title: test.title, wallClockStartedAt: test.wallClockStartedAt.getTime() })
+      await Cypress.backend('protocol:test:before:run:async', { id: attr.id, attempt: attr.currentRetry + 1, timestamp: attr.wallClockStartedAt.getTime() })
```

Comment:
why are we incrementing the attempt by 1?

Reply:
I think the use-case is slightly different. In the app, we are actually running the test, so we are dealing with retries (e.g. should I retry this test and what is the current retry) whereas in Test Replay, we are asking what attempt is this. With that said, I'm not opposed to changing this to `currentRetry` to match the app (though it feels odd to say "current" when all of it is in the past).

Reply:
Updated to `currentRetry`: [b4b0dea](https://github.com/cypress-io/cypress/pull/26216/commits/b4b0dea1bed69ee9f8931389b1968b0c9312c77b)